src/diffusers/loaders/ip_adapter.py

```diff
@@ -46,8 +48,8 @@ class IPAdapterMixin:
     def load_ip_adapter(
         self,
         pretrained_model_name_or_path_or_dict: Union[str, Dict[str, torch.Tensor]],
-        subfolder: str,
         weight_name: str,
+        subfolder: Optional[str] = None,
```

Comment:
We can pass this to the `kwargs` right? I don't think there's a need to expose this specifically. WDYT?

Reply:
> We can pass this to the `kwargs` right? I don't think there's a need to expose this specifically. WDYT?

Yes, we can pass it as keyword argument. The other IP Adapter models need the image encoder, stored in a subfolder of H94/IP-Adapter, while FaceID model doesn't require an image encoder, so I made it Optional. I will remove it in the next update